Makefile

```diff
@@ -198,10 +202,86 @@ db-reset:
 libzauth:
 	$(MAKE) -C libs/libzauth install
 
+#################################
 # Useful when using Haskell IDE Engine
 # https://github.com/haskell/haskell-ide-engine
 #
 # Run this again after changes to libraries or dependencies.
 .PHONY: hie.yaml
 hie.yaml:
 	stack exec gen-hie > hie.yaml
+
+#####################################
+# Today we pretend to be CI and run integration tests on kubernetes
+# (see also docs/developer/processes.md)
+#
+# NOTE: This uses local helm charts from .local/charts (which it builds before running this)
+#
+# NOTE/WARNING: By default, it uses local docker image tags,
+# which will not work at this time on your remote kubernetes cluster. [FUTUREWORK: local kubernetes cluster]
+#
+# If you wish to use docker images that are uploaded to quay.io, you must set DOCKER_TAG
+#
+#   DOCKER_TAG=<desired-wire-server-docker-tag> make kube-integration
+#
+# and if you don't know what a good DOCKER_TAG might be, you can run
+#
+#   make latest-brig-tag
+#
+# This task requires: [FUTUREWORK: add tooling setup to wire-server]
+#   - helm (version 3.1.1)
+#   - kubectl
+#   - a valid kubectl context configured (i.e. access to a kubernetes cluster)
+.PHONY: kube-integration
+kube-integration: charts
+	# by default "test-<your computer username> is used as namespace
+	export NAMESPACE=test-$(USER); ./hack/bin/integration-setup.sh
+	export NAMESPACE=test-$(USER); ./hack/bin/integration-test.sh
+
+.PHONY: kube-integration-teardown
+kube-integration-teardown:
+	export NAMESPACE=test-$(USER); ./hack/bin/integration-teardown.sh
+
+.PHONY: latest-brig-tag
+latest-brig-tag:
+	./hack/bin/find-latest-docker-tag.sh
+
+.PHONY: release-chart-%
+release-chart-%:
+	if [ "${HELM_SEMVER}" = "0.0.42" ]; then \
+	      echo "Environment variable HELM_SEMVER not set to non-default value"; \
+	    exit 1; \
+	fi
+	if [ "${DOCKER_TAG}" = "local" ]; then \
+	      echo "Environment variable DOCKER_TAG not set to non-default value"; \
+	    exit 1; \
+	fi
+	make chart-$(*)
+
+.PHONY: chart-%
+chart-%:
+	./hack/bin/copy-charts.sh $(*)
+	./hack/bin/set-wire-server-image-version.sh $(DOCKER_TAG)
```

Comment:
Defaulting to `local` for `DOCKER_TAG` is hazardous due to race conditions among ourselves. Maybe we can use `$USER`?

Reply:
Good suggestion, thanks!